docs/Common-Issues-for-Core-Contributors.md

```diff
@@ -6,6 +6,12 @@ This is a page for maintainers to diagnose certain build errors.
 
 ## Issues
 
+### Bottle publishes failed but the commits are correct in the git history
+
+Follow these steps to fix this issue:
+* `git reset --hard origin/master` to reset to the latest commit before the bottle publish.
```

Comment:
This reset should be done to the commit before all the commits created by `brew pr-pull`.